src/whisper_ctranslate2/diarization.py

```diff
@@ -32,14 +36,12 @@ def unload_model(self):
         torch.cuda.empty_cache()
 
     def _load_model(self) -> "Pipeline":
-        model_name = "pyannote/speaker-diarization-3.1"
+        model_name = "pyannote/speaker-diarization-community-1"
         device = torch.device(self.device)
-        model_handle = Pipeline.from_pretrained(
-            model_name, token=self.use_auth_token
-        )
+        model_handle = Pipeline.from_pretrained(model_name, token=self.use_auth_token)
         if model_handle is None:
             raise ValueError(
-                f"The token Hugging Face token '{self.use_auth_token}' for diarization is not valid or you did not accept the EULA"
+                f"The token Hugging Face token '{self.use_auth_token}' for diarization is not valid or you did not accept the EULAs for the necessary models. See https://github.com/Softcatala/whisper-ctranslate2#diarization-speaker-identification"
```

Comment:
concern: the text at this link still refers to accepting the terms for `pyannote/speaker-diarization-3.1` and this PR switches to `pyannote/speaker-diarization-community-1`